psalm-baseline.xml

```diff
@@ -1214,6 +1233,9 @@
     </RedundantConditionGivenDocblockType>
   </file>
   <file src="test/Service/ContainerAbstractServiceFactoryTest.php">
+    <DeprecatedClass occurrences="1">
+      <code>new Config</code>
+    </DeprecatedClass>
```

Comment:
This class is no longer used? Why is the class listed here?

Reply:
Resolved